.gitlab/e2e_pre_test.yml

```diff
@@ -5,7 +5,10 @@ e2e_pre_test:
     - !reference [.on_e2e_changes_or_manual]
   stage: e2e_pre_test
   extends: .new_e2e_template
+  needs: []
   script:
     - inv -e new-e2e-tests.run --targets ./test-infra-definition --junit-tar "junit-${CI_JOB_NAME}.tgz" ${EXTRA_PARAMS}
   variables:
     TEAM: "agent-platform"
+    # override to use latest stable agent
+    E2E_PIPELINE_ID: ""
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ The variable name here could be more obviously connected to the stable agent? At surface value it doesn't make a ton of sense to me.

Reply:
In the template we set this variable to the `CI_PIPELINE_ID`. By overriding it to empty, it will use the default agent version, being latest stable. If I set the pipeline id to the pipeline that built latest stable, I would need to update it manually at each release. It is not ideal, we could have a value saying use `stable` or `pipeline` or `rc` or `beta`, a channel option. Still we would need to override the template `E2E_PIPELINE_ID` value